lib/smart_proxy_openscap/openscap_api.rb

```diff
@@ -14,6 +14,7 @@ module Proxy::OpenSCAP
   class Api < ::Sinatra::Base
     include ::Proxy::Log
     helpers ::Proxy::Helpers
+    authorize_with_trusted_hosts
 
     put "/arf/:policy" do
```

Comment:
I'd also choose POST also for here, but do not want to refactor this atm.
policies are configured on the client host via puppet.
[foreman_scap_client](https://github.com/OpenSCAP/foreman_scap_client) runs the policies, generates reports and [PUTS the reports to the proxy ](https://github.com/OpenSCAP/foreman_scap_client/blob/master/lib/foreman_scap_client/client.rb#L91)